app/src/main/java/fr/neamar/kiss/MainActivity.java

```diff
@@ -323,6 +325,18 @@ public boolean onItemLongClick(AdapterView<?> parent, View v, int pos, long id)
 
         UiTweaks.updateThemePrimaryColor(this);
         UiTweaks.tintResources(this);
+
+        // Transparent Search and Favorites bar
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
+            TypedValue typedValue = new  TypedValue();
+            getTheme().resolveAttribute(R.attr.listBackgroundColor, typedValue, true);
```

Comment:
This can be an issue when using the "light theme" (default) and "dark theme", since the `listBackgroundColor` isn't transparent then.

In this case, it's probably easier to just use "android.transparent" directly?